pytorch_lightning/lite/wrappers.py

```diff
@@ -11,58 +11,24 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from typing import Any, Callable, Dict, Generator, Iterator, Optional, Union
+from typing import Any, Dict, Generator, Iterator, Optional, Union
 
 import torch
 from torch import nn as nn
 from torch import Tensor
-from torch.optim import Optimizer
 from torch.utils.data import DataLoader
 
 from pytorch_lightning.core.mixins import DeviceDtypeModuleMixin
+from pytorch_lightning.core.optimizer import LightningOptimizer
 from pytorch_lightning.plugins import PrecisionPlugin
-from pytorch_lightning.strategies import Strategy
 from pytorch_lightning.utilities.apply_func import apply_to_collection, move_data_to_device
 
 
-def _do_nothing_closure() -> None:
-    return None
-
-
-class _LiteOptimizer:
-    def __init__(self, optimizer: Optimizer, strategy: Strategy) -> None:
-        """LiteOptimizer is a thin wrapper around the :class:`~torch.optim.Optimizer` that delegates the optimizer
-        step calls to the strategy plugin.
-
-        The underlying wrapped optimizer object can be accessed via the property :attr:`optimizer`.
-
-        Args:
-            optimizer: The optimizer to wrap
-            strategy: Reference to the strategy for handling the optimizer step
-        """
```

Comment:
can we keep the docstring please? I really think the one from LightningOptimizer sucks.